test/test_torch.py

```diff
@@ -12491,6 +12711,14 @@ def test_lerp_mem_overlap(self):
         with self.assertRaisesRegex(RuntimeError, 'single memory location'):
             start.lerp_(end, weight)
 
+    @unittest.expectedFailure
+    def test_pow_mem_overlap(self):
+        TestTorch.binary_check_mem_overlap(self, 'pow_', device='cpu')
+        tensor = torch.tensor(value, device=device).expand(3, 3)
```

Comment:
Everything below `TestTorch.binary_check_mem_overlap(self, 'pow_', device='cpu')` need to be removed ( already done inside of the `def binary_check_mem_overlap(self, inplace_op, value=-0.5, device='cpu'):`

Reply:
binary_check_mem_overlap does tensor^tensor, I wanted to check tensor^scalar